package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "currency-converter",
+  "description": "This is a currency to currency module created in NodeJS. It's purpose is to take one currency format and convert it to another currency and return the result.",
+  "version": "1.0.1",
+  "main": "currencyConverter.js",
+  "dependencies": {
+    "request": "*",
```

Comment:
specify a version for these two dependencies. You can get the latest version number from `npmjs.com`